astropy/utils/data.py

```diff
@@ -70,8 +70,12 @@ class Conf(_config.ConfigNamespace):
     remote_timeout = _config.ConfigItem(
         10.,
         'Time to wait for remote data queries (in seconds). Set this to zero '
-        'to prevent any attempt to download anything.',
+        'to prevent any attempt to download anything (this will stop working '
+        'in a future release, use allow_internet=False instead).',
         aliases=['astropy.coordinates.name_resolve.name_resolve_timeout'])
+    allow_internet = _config.ConfigItem(
+        True,
+        'If False, prevents any attempt to download from Internet.')
```

Comment:
Should there be an explanation for how this relates to the other option that restricts internet activity? The one we first thought might do what this one does? (Sorry don't remember the name right now.)

Reply:
The one we first discussed, I think, is something in https://github.com/astropy/pytest-remotedata . Even that pytest plugin does not have have specific `allow_internet` function. I will consider another name if there is a better suggestion.